odbc/src/tests.rs

```diff
@@ -1,10 +1,14 @@
 use crate::{
+    errors::ODBCError,
```

Comment:
I think we need some integration tests as well to have complete coverage. Now that @nbagnard 's test framework PR has been merged, this is now possible!

In particular, I think we should test the following:
- Behavior of the newly implemented ODBC API functions
- Do error lists get concatenated or replaced when the same handle is used for consecutive function calls that return an error.

If you'd prefer to defer this to a separate ticket, that's fine by me as well

Reply:
Filed [SQL-663](https://jira.mongodb.org/browse/SQL-663)